controllers/pattern_controller.go

```diff
@@ -237,11 +260,28 @@ func (r *PatternReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	if err := r.postValidation(qualifiedInstance); err != nil {
 		return r.actionPerformed(qualifiedInstance, "validation", err)
 	}
+
+	// Update CR if necessary
+	var fUpdate bool
+	fUpdate, err = r.updatePatternCRDetails(qualifiedInstance)
+
+	if err == nil && fUpdate {
+		log.Printf("\x1b[32;1m\tPattern CR Updated\x1b[0m\n")
+	}
+
```

Comment:
Maybe just always call `actionPerformed()`?
I imagine that `lastReconcileTime` changes often enough that we're not saving much processing (and we're performing a bunch to calculate if something changed).

Reply:
Oh, you're filtering out updates where only `lastReconcileTime` changed.
I don't think it's a good idea to have `lastReconcileTime` but for it to contain stale data.  
Either we should remove it, call it something that matches how we're updating it, or update it every time.